docs/api.yaml

```
@@ -8701,6 +8703,22 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error403'
+        409:
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                required:
+                - code
+                type: object
+                properties:
+                  code:
+                    type: string
```

```suggestion
                    type: number
```

Some related discussion here: https://github.com/getodk/central-backend/pull/1133#issuecomment-2083947231